### ISSUE TITLE
Explore enhancement - Add State of Bitshares Network tab

### ISSUE THREAD

**Issue by TechsUsInc** [bitshares/bitshares-ui#2402]
**Is your feature request related to a problem? Please describe.**
The number of platforms being built on top of Bitshares has and will continue to increase rapidly. Jumping from one platform to the next can be a chore, and even un-secure just trying to make comparisons or exchange. 

**Describe the solution you'd like**

![181212-explore_enhancement_1](https://user-images.githubusercontent.com/18186497/50949605-7a285d80-146c-11e9-91a2-654ebf572f3b.JPG)
Current
![181212-explore_enhancement_2](https://user-images.githubusercontent.com/18186497/50949885-a1cbf580-146d-11e9-913e-8509effdccff.jpg)
 Proposed
![181212-explore_enhancement_3](https://user-images.githubusercontent.com/18186497/50949710-e7d48980-146c-11e9-91a8-b838d8121efd.jpg) Favored
Clicking the tab takes the user to a list (similar to CryptoKong's Ecosystem List but looks like the GIF below ) which entails projects built on Bitshares providing any user a one page view of real time trading. It would be nice to incorporate both send and receive broadcasts but maybe not possible.
 
Each project could be click-able taking the user to that DEX, website, discord, etc or all.

If for nothing else, it provides an impressive network activity report for evaluation, comparison, usage, and other metrics.
![coinlib_mv](https://user-images.githubusercontent.com/18186497/50950254-08054800-146f-11e9-9e33-1853c8dc4d27.gif)

**Additional context**
GIF captured from coinlib.io

**Comment by grctest** [comment]
Where would this data be sourced? It seems more of a "State of BTS centralized exchange trading volumes" than a state of the network, which was a successful news article series on steemit.

**Comment by TechsUsInc** [comment]
That's a good question, requires more thought. Basically I was suggesting a Bitshares Ecosystem panel in real time (similar to above image) mindful of credit to the state of the network as you said, Fuzzy, and others that contributed towards that effort if it's even possible given so many variables.

**Comment by TechsUsInc** [comment]
@grctest I found these open source tools today if you are interested.
 Please take a look at https://blockmodo.com/quotes/EOS/markets

1.Released yesterday was developed with EOS in mind streaming real time pulse including 
   market,social,and dev activity.
   Blockmodo EOS-Tab:  https://github.com/Blockmodo/eos_tab 
2. Blockmodo Coin-registry:  https://github.com/Blockmodo/coin_registry
3. CCXT-A JavaScript / Python / PHP cryptocurrency trading library with support for more than 130 bitcoin/altcoin exchanges and is used to connect and trade with cryptocurrency / altcoin exchanges and payment processing services worldwide. It provides quick access to market data for storage, analysis, visualization, indicator development, algorithmic trading, strategy backtesting, bot programming, webshop integration and related software engineering. It is intended to be used by coders, developers, technically-skilled traders, data-scientists and financial analysts for building trading algorithms on top of it. (Bitshares should be on this list IMO)
CCXT: https://github.com/ccxt/ccxt

**Comment by startailcoon** [comment]
This is more of a "Trading state of BTS" than a state of the "Network". Cool idea, but not sure if I like it inside the client. It would also require a large amount of external requests to get the data, not sure everyone would like that.

`Needs More Discussion`

**Comment by TechsUsInc** [comment]
@startailcoon Agreed that it would be taxing on the client and may be could be used elsewhere.
